app/models/form_profile.rb

```diff
@@ -126,7 +147,7 @@ class FormProfile
     '21-686C' => ::FormProfiles::VA21686c,
     '21P-0516-1-UPLOAD' => ::FormProfiles::FormUpload,
     '21P-0518-1-UPLOAD' => ::FormProfiles::FormUpload,
-    '21P-527EZ' => ::FormProfiles::VA21p527ez,
+    '21P-527EZ' => form_class_namespace('pension')::VA21p527ez,
```

Comment:
I think this constant will only get calculated when the class is first created, and not on each run (meaning that changes in the flipper status won't impact it until the API refreshes).
I ran `FormProfile.for(form_id: '21P-527EZ',user: nil)` in the rails console with the flipper off, then turned it on (and confirmed in the console it was `true`) and each time received `FormProfiles::VA21p527ez`. Then I closed and restarted the console and received `Pensions::FormProfiles::VA21p527ez`.